backend/clubs/views.py

```diff
@@ -2790,6 +2790,165 @@ def create_tickets(self, request, *args, **kwargs):
 
         return Response({"detail": "Successfully created tickets"})
 
+    @action(detail=True, methods=["post"])
+    @transaction.atomic
+    @update_holds
+    def issue_tickets(self, request, *args, **kwargs):
```

Comment:
Please lock behind same perms for `create_tickets`.

Reply:
thanks for catching